.github/CONTRIBUTION_PROCESS.md

```diff
@@ -0,0 +1,37 @@
+This repository contains the developer documentation sources.
+
+# Reviewers
+
+On this reposiroty we have primary reviewers and regular reviewers.
+
+Primary reviewers are the people who mostly shape and structure this repository. Although it's not mandatory, it's highly recommended that at least one of them review and approve a Pull Request before he can be merged.
```

Comment:
```suggestion
Primary reviewers are the people who mostly shape and structure this repository. Although it's not mandatory, it's highly recommended that at least one of them review and approve a Pull Request before it can be merged.
```